### ISSUE TITLE
smart highlighting is not so smart

### ISSUE THREAD

**Issue by rhelmot** [angr/angr-management#198]
![image](https://user-images.githubusercontent.com/2498805/102698490-f9973d80-41fa-11eb-8bfa-c3debe686f53.png)

I'm guessing it doesn't highlight the second variable usage because it views the assignment as killing the previous value, but the first one is an access by reference, so it should apply to all usages of the variable in this function, regardless of where it gets killed.

**Comment by ltfish** [comment]
`s_10` at 0x48cb2 is a new def regardless of how `s_10` was used previously. At 0x48ca9, `s_10` is a use of the previous def.

You may switch to less-smart highlighting by pressing `a`.

**Comment by rhelmot** [comment]
right... but 0x48ca9 isn't strictly a use of the variable. it's a reference which may be used as many times in the future as it cares to be, including after the current def is killed. Intuitively, I would like to see that reflected in the highlighting
